src/java.base/share/classes/java/net/NetworkInterface.java

```diff
@@ -97,6 +117,11 @@ public String getName() {
      * {@link NetPermission}("getNetworkInformation") permission, then all
      * InetAddresses are returned.
      *
+     * @implNote
+     * The returned enumeration contains all or a subset of the InetAddresses that were
```

Comment:
perhaps additional comma for emphasis 
The returned enumeration contains all, or a subset, of the InetAddresses . . .